PiwikTracker/PiwikTracker.swift

```diff
@@ -38,10 +39,13 @@ final public class PiwikTracker: NSObject {
     ///   - siteId: The unique site id generated by the server when a new site was created.
     ///   - queue: The queue to use to store all analytics until it is dispatched to the server.
     ///   - dispatcher: The dispatcher to use to transmit all analytics to the server.
-    required public init(siteId: String, queue: Queue, dispatcher: Dispatcher) {
+    required public init(siteId: String, queue: Queue, dispatcher: Dispatcher, suiteName: String? = nil) {
```

Comment:
Hi @minitrue22, thanks for the feedback. I've been thinking about just using the siteId but there is a downside to it. If one wants to use different Serves and the siteIds on both are different that would lead to a conflict. What do you think about using the siteId and the dispatchers-url in combination for the suiteName? Downside of this would be: What if one changes the URL of the Server or such? That would change all those values.